app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,10 @@
+.card {
+  height: 360px;
+  background-color: $light-pink;
+  color: $black;
+}
+
+.btn:hover {
+  text-decoration: none !important;
```

Comment:
Why are you exactly overwriting this? And why in the _card.scss file?

Reply:
Your right, I was experimenting with removing the "link" styling, but we don't need it. I took it out.